plugins/BEdita/API/config/routes.php

```diff
@@ -22,6 +22,11 @@
     ],
     function (RouteBuilder $routes) {
         $routes->connect(
+            '/home',
+            ['controller' => 'Home', 'action' => 'index'],
+            ['_method' => 'GET', '_name' => 'home:index']
```

Comment:
Hence I the home document is actually some sort of meta resource, I'd go for `home` alone, omitting the `:index` suffix.